ci-operator/step-registry/cucushift/hypershift-extended/external-oidc/enable/cucushift-hypershift-extended-external-oidc-enable-commands.sh

```diff
@@ -59,6 +68,19 @@ yq-v4 'select(.kind == "HostedCluster") *= load("/tmp/patch.yaml")' "${SHARED_DI
 
 echo "Applying patched artifacts"
 oc apply -f "${SHARED_DIR}"/hypershift_create_cluster_render_ext_oidc_enabled.yaml
+# TODO HOSTED_CLUSTER_VERSION is a placeholder for now, how to get its value?
+if [[ $(awk "BEGIN {print ($HOSTED_CLUSTER_VERSION >= 4.18)}") == "1"  ]]; then
```

Comment:
@heliubj18 do you know to get value of HOSTED_CLUSTER_VERSION? I'm not sure whether `RELEASE_IMAGE=${HYPERSHIFT_HC_RELEASE_IMAGE:-$RELEASE_IMAGE_LATEST}` uses tag format or sha256 digest format.

Reply:
Resolved the question now using `oc image info`

Reply:
for a running hosted cluster